src/main/java/com/commercetools/sync/sdk2/commons/BaseSync.java

```diff
@@ -27,7 +29,7 @@ protected BaseSync(@Nonnull final U statistics, @Nonnull final V syncOptions) {
    *     instance of {@code U} which is a subclass of {@link BaseSyncStatistics} representing the
```

Comment:
Please also update the generics in the comments.

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/942/commits/394405f3780b2c758a49607183f3e63176938c99